internal/pkg/cli/svc_package.go

```diff
@@ -296,6 +296,8 @@ func (o *packageSvcOpts) getSvcTemplates(env *config.Environment) (*wkldCfnTempl
 		interpolator: o.newInterpolator(o.appName, o.envName),
 		ws:           o.ws,
 		unmarshal:    o.unmarshal,
+		targetEnv:    o.targetEnv,
+		svcType:      "",
```

Comment:
If we move `IsServiceAvailableInRegion` to the `deployer` we probably wouldn't need to pass around `svcType` anymore